src/en/howto-applicationupdateseries.md

```diff
@@ -0,0 +1,117 @@
+Title: Change an application's series pre-juju2.3
+
+#  Change an application's series pre-juju2.3
+
+## Overview
+
+When deploying an application with juju you can specify the series it
+will use, or use the default series for the model.  Prior to juju 2.3,
+the application would always use the series specified at deployment
+when adding units with no way to change it.
+
+With juju 2.3 or later it's possible let juju know you'd like to add
+units with a different series moving forward by running the command:
+`juju update-series <application> <series>`
+
+Here we'll cover the manual steps on how to do this pre-juju2.3.
+
+## Prerequisites
+
+1. Check that the current charm revision in use for the application supports the
+series you wish to use in the future.  This checked on in the juju charm store.
+
+2. Name of the application.
+
+3. Name of the model where the application lives.
+
+4. Admin privilages to the controller where the model lives.
+
+## Update
+
+Start by switching to the juju controller where the changes will happen using
```

Comment:
Juju controller

Reply:
why is juju capitalized here and not other places?